script/main_test.py

```diff
@@ -1,5 +1,27 @@
 import unittest
 import main
+import sqlite3
+
+
+def create_connection(db_file):
+  conn = None
+  try:
+    conn = sqlite3.connect(db_file)
+  except sqlite3.Error as e:
+    print(e)
+  
+  return conn
+
+
+class TestMostPopularCities(unittest.TestCase):
+
+  def setUp(self):
```

Comment:
Very good.

Please also add function `tearDown()` to close this connection.
https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown